packages/api/types.ts

```diff
@@ -19,90 +156,524 @@ import type {
+        response: SellContractResponse;
+    };
+    set_account_currency: {
+        request: SetAccountCurrencyRequest;
+        response: SetAccountCurrencyResponse;
+    };
+    set_financial_assessment: {
+        request: SetFinancialAssessmentRequest;
+        response: SetFinancialAssessmentResponse;
+    };
+    set_self_exclusion: {
+        request: SetSelfExclusionRequest;
+        response: SetSelfExclusionResponse;
+    };
+    set_settings: {
+        request: SetAccountSettingsRequest;
+        response: SetAccountSettingsResponse;
+    };
+    statement: {
+        request: StatementRequest;
+        response: StatementResponse;
+    };
+    states_list: {
+        request: StatesListRequest;
+        response: StatesListResponse;
+    };
+    ticks_history: {
+        request: TicksHistoryRequest;
+        response: TicksHistoryResponse;
     };
     ticks: {
         request: TicksStreamRequest;
         response: TicksStreamResponse;
     };
-    new_account_virtual: {
-        request: NewVirtualMoneyAccountRequest;
-        response: NewVirtualMoneyAccountResponse;
+    time: {
+        request: ServerTimeRequest;
+        response: ServerTimeResponse;
     };
-    get_account_status: {
-        request: AccountStatusRequest;
-        response: AccountStatusResponse;
+    tnc_approval: {
+        request: TermsAndConditionsApprovalRequest;
+        response: TermsAndConditionsApprovalResponse;
+    };
+    topup_virtual: {
+        request: TopUpVirtualMoneyAccountRequest;
+        response: TopUpVirtualMoneyAccountResponse;
+    };
+    trading_durations: {
+        request: TradingDurationsRequest;
+        response: TradingDurationsResponse;
+    };
+    trading_platform_investor_password_reset: {
+        request: TradingPlatformInvestorPasswordResetRequest;
+        response: TradingPlatformInvestorPasswordResetResponse;
+    };
+    trading_platform_password_reset: {
+        request: TradingPlatformPasswordResetRequest;
+        response: TradingPlatformPasswordResetResponse;
+    };
+    trading_servers: {
+        request: ServerListRequest;
+        response: ServerListResponse;
+    };
+    trading_times: {
+        request: TradingTimesRequest;
+        response: TradingTimesResponse;
+    };
+    transaction: {
+        request: TransactionsStreamRequest;
+        response: TransactionsStreamResponse;
+    };
+    transfer_between_accounts: {
+        request: TransferBetweenAccountsRequest;
+        response: TransferBetweenAccountsResponse;
+    };
+    unsubscribe_email: {
+        request: UnsubscribeEmailRequest;
+        response: UnsubscribeEmailResponse;
+    };
+    verify_email_cellxpert: {
+        request: VerifyEmailCellxpertRequest;
+        response: VerifyEmailCellxpertResponse;
+    };
+    verify_email: {
+        request: VerifyEmailRequest;
+        response: VerifyEmailResponse;
+    };
+    website_status: {
+        request: ServerStatusRequest;
+        response: ServerStatusResponse;
```

Comment:
nice :D

Reply:
I stole it from @mohsen-deriv 😁

Reply:
lol, I stole the whole thing from you man :smile: